client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+
+export default React.createClass( {
+	displayName: 'BluehostStoreStep',
+
+	redirectToPartner() {
+		if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+			window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
+		} else {
+			window.location.href = 'https://www.bluehost.com/wordpress';
+		}
+	},
+
+	getPrice() {
+		return ( 'bluehost' === abtest( 'signupStoreBenchmarking' ) ) ? '$3.95' : '$12.95';
```

Comment:
We should check if we need to display other `EN` locale prices, like GBP and AUD